LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
It's a pain but please do a seperate commit for the license. No need for approval.

Reply:
Move them to separate commit and PR https://github.com/giantswarm/k8s-endpoint-updater/pull/24